content/01-definitions/definitions.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Definitions"
+icon: "ti-search"
```

Comment:
This line you only need for an `_index.en.md` file. In the index file, you use that to initialize and create the category. Without an index page, your category will not appear on the site homepage. This page here will publish as `https://unicef.github.io/ooi-toolkit-ds/01-definitions/definitions/`

Reply:
Do you suggest I remove this descriptive annotation from the `definitions.md` file? Should I create a separate index file?

Reply:
The description annotation is useful. This is used for OpenGraph data, or what pops up in the box when you share a link on social media or a messaging app. It should be a sentence or two summary of the article contents.